README.md

```diff
@@ -91,9 +91,18 @@ cd /tmp/butido-test-repo
 | tree        | The tree structure that is computed before a packages is built to find out all (transitive) dependencies         |
 
 
+# Authors
```

Comment:
Is this fine or should we, e.g., call it `Authors and maintainers`?

Reply:
how about this:

current maintainers: Mi + Ni
original author: Ma
(full names/email of course)

Reply:
Your proposal isn't clear to me. What exact changes do you suggest? Do you mean that I should simply drop "Passive maintainers"?

Personally, I'd like to list Erdmut and you as well since you both have admin/write/commit access and could be pinged/consulted for PRs, issues, etc (incl. the ability to merge PRs, label/handle issues, etc.). So you're basically still maintainers - you just don't do any development yourself (which is why I used "passive" but that word isn't ideal as it's meaning is too close to "inactive", etc. - not sure if "Project managers", "Owners", "Primary users", or something like that could be better).

There are many different definitions of what a maintainer is but in this case I like the one from  https://opensource.guide/leadership-and-governance/:
> For some projects, “maintainers” are the only people in a project with commit access. In other projects, they’re simply the people who are listed in the README as maintainers.
>
> A maintainer doesn’t necessarily have to be someone who writes code for your project. It could be someone who’s done a lot of work evangelizing your project, or written documentation that made the project more accessible to others. Regardless of what they do day-to-day, a maintainer is probably someone who feels responsibility over the direction of the project and is committed to improving it.